openstef/model/regressors/dazls.py

```diff
@@ -103,14 +103,20 @@ def fit(self, features, target):
         )
         self.adaptation_model.fit(adaptation_model_input, y_train)
 
-    def predict(self, x: np.array):
+    def predict(self, x: np.array, return_sub_preds=False):
```

Comment:
Nice idea to add this argument!

Could you add a type_hint: `return_sub_preds: bool = False`.